src/scss/components/_header.scss

```diff
@@ -9,7 +9,7 @@
   @include container();
   align-items: center;
   display: flex;
-  height: rhythm(6);
+  height: rhythm(8);
```

Comment:
Looks like we need to update the `top` value of the expanded mobile list to match the new header height.

<img width="1680" alt="Screen Shot 2020-12-27 at 12 30 35 PM" src="https://user-images.githubusercontent.com/411808/103178367-6afc6f00-483f-11eb-9ba7-dc72e245c1f0.png">

Reply:
On it - good catch! Sorry I missed it.